lib/elixir_auth_github.ex

```diff
@@ -1,133 +1,42 @@
 defmodule ElixirAuthGithub do
   @moduledoc """
-  ElixirAuthGithub is a module to help you with Github OAuth in Elixir and Phoenix.
+  ElixirAuthGithub is an Elixir package that handles all your GitHub OAuth needs
+  so you can add "Sign in with GitHub" to any Elixir/Phoenix App.
 
-  We created it because everyone at dwyl uses github (including our clients!) so github OAuth makes sense for our internal (and external) tools. As a result, there's no use reinventing the wheel every project, and by making it into a module we can help other people as well!
```

Comment:
This is quite a lot of text to include in the module.
I prefer to have it in the `README.md` which is where I _hypothesize_ more people will _discover/read_ it.
If you _disagree_ please comment! 👍